browser/app/less/inc/alert.less

```diff
@@ -36,33 +31,32 @@
     }
 
     &.alert-info {
-        background: @blue;
+        background: @light-blue;
     }
 
-    @media(max-width: (@screen-xs-max)) {
+    @media (max-width: (@screen-xs-max)) {
         left: 20px;
         width: ~"calc(100% - 40px)";
         max-width: 100%;
     }
 
-    .progress {
-        margin: 10px 10px 8px 0;
-        height: 5px;
-        box-shadow: none;
-        border-radius: 1px;
-        background-color: @blue;
-        border-radius: 2px;
-        overflow: hidden;
-    }
-
-    .progress-bar {
-        box-shadow: none;
-        background-color: @white;
-        height: 100%;
-    }
-
     .close {
```

Comment:
It's loaded slightly different in my Firefox(54.0.1).

![alert](https://user-images.githubusercontent.com/13393018/28633473-8dbb790e-7251-11e7-8a5c-a3fe10815341.png)

Still, it's not centered properly. I will work on it.